app/dashboard/templates/bounty/details2.html

```diff
@@ -1163,6 +1163,11 @@ <h3>{{ noscript.keywords }}</h3>
       <script src="{% static "v2/js/lib/tezos/taquito-beacon-wallet.min.js" %}"></script>
       <script src="{% static "v2/js/pages/bounty_detail/tezos_extension.js" %}"></script>
 
+    {% elif web3_type == 'casper_ext' %}
+
+      <script src="{% static "v2/js/lib/casper/casper_js_sdk.js" %}"></script>
```

Comment:
@gdixon this is right with the new yarn config?

Reply:
yup yup ! i asked the same thing in the PR :P all custom lib cross chain file continue to live in the path